cmd/natsresolverconfigbuilder.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2018-2019 The NATS Authors
```

Comment:
Update copyright year - on all files that were modified.

Reply:
changed to Copyright 2018-2020